src/js/webrequest.js

```diff
@@ -536,7 +536,7 @@ function checkAction(tabId, url, quiet, frameId){
   if (action && ! quiet) {
     badger.logTrackerOnTab(tabId, requestHost, action);
     if (_.contains(constants.BLOCKED_ACTIONS, action)) {
-      badger.updateCount(tabId);
+      setTimeout(badger.updateCount(tabId), 0);
```

Comment:
`updateCount` is immediately executed, returning a string, which is then passed into `setTimeout`. You need to do something like the following instead:
```javascript
setTimeout(function () {
  badger.updateCount(tab_id);
}, 0);
```